docker/entrypoint.sh

```diff
@@ -289,6 +289,11 @@ then
       /bin/sed -i "s/tls_starttls .*$/tls_starttls off/" /etc/msmtprc
     fi
 
+    if [ "$CMS_SMTP_CHECK_TLS_CERTIFICATE" == "NO" ]
+    then
+      /bin/sed -i "s/tls .*$/tls_nocertcheck/" /etc/msmtprc
```

Comment:
I think this will match the line:

```
tls on
```

and replace it with `tls_nocertcheck`?

But I think the correct syntax would be to have both? ie

```
tls on
tls_nocertcheck
```

Perhaps it might be better to have 

```
# CMS_SMTP_USE_TLS
tls on
#CMS_SMTP_CHECK_TLS_CERTIFICATE
#tls_nocertcheck
```

and then update the sed command to remove the comment if necessary?

Reply:
Yes, absolutly. Both options are necessary :
```
tls on
tls_nocertcheck
```
You are right Alex.